_studio/shared/umc/codec/mpeg2_dec/src/umc_mpeg2_va_packer.cpp

```diff
@@ -26,6 +26,7 @@
 #include "umc_mpeg2_slice.h"
 #include "umc_mpeg2_frame.h"
 #include "umc_mpeg2_va_packer.h"
+#include "mfx_session.h"
```

Comment:
Better to avoid such dependency and send priority as a parameter from component level